crates/cxx-qt-gen/src/generator/cpp/property/mod.rs

```diff
@@ -32,19 +32,27 @@ pub fn generate_cpp_properties(
         let cxx_ty = syn_type_to_cpp_type(&property.ty, type_names)?;
 
         generated.metaobjects.push(meta::generate(&idents, &cxx_ty));
-        generated
-            .methods
-            .push(getter::generate(&idents, &qobject_ident, &cxx_ty));
-        generated
-            .private_methods
-            .push(getter::generate_wrapper(&idents, &cxx_ty));
-        generated
-            .methods
-            .push(setter::generate(&idents, &qobject_ident, &cxx_ty));
-        generated
-            .private_methods
-            .push(setter::generate_wrapper(&idents, &cxx_ty));
-        signals.push(signal::generate(&idents, qobject_idents));
+
+        if let Some(getter) = getter::generate(&idents, &qobject_ident, &cxx_ty) {
+            generated.methods.push(getter);
+            generated
+                .private_methods
+                .push(getter::generate_wrapper(&idents, &cxx_ty));
+        }
```

Comment:
in theory you could add a else case here that is unreachable to ensure it doesn't happen?
```rust
} else {
    unreachable!("QProperty should always have a getter");
}
```

Reply:
The if let block is for generating the method, so the else case would basically be catching the getter not being generated (custom getter provided) so it's not an error to return None from generate here

Reply:
cool, done